tests/test_codebase/test_mmdet3d/data/model_cfg.py

```diff
@@ -228,3 +228,92 @@
             post_max_size=83,
             nms_thr=0.2)))
+voxel_size = [0.25, 0.25, 8]
+pointpillars_nus_model = dict(
+    pts_voxel_layer=dict(
+        max_num_points=64,
+        point_cloud_range=[-50, -50, -5, 50, 50, 3],
+        voxel_size=voxel_size,
+        max_voxels=(30000, 40000)),
+    pts_voxel_encoder=dict(
+        type='HardVFE',
+        in_channels=4,
+        feat_channels=[64, 64],
+        with_distance=False,
+        voxel_size=voxel_size,
+        with_cluster_center=True,
+        with_voxel_center=True,
+        point_cloud_range=[-50, -50, -5, 50, 50, 3],
+        norm_cfg=dict(type='naiveSyncBN1d', eps=1e-3, momentum=0.01)),
+    pts_middle_encoder=dict(
+        type='PointPillarsScatter', in_channels=64, output_shape=[400, 400]),
+    pts_backbone=dict(
+        type='SECOND',
+        in_channels=64,
+        norm_cfg=dict(type='naiveSyncBN2d', eps=1e-3, momentum=0.01),
+        layer_nums=[3, 5, 5],
+        layer_strides=[2, 2, 2],
+        out_channels=[64, 128, 256]),
+    pts_neck=dict(
+        type='FPN',
+        norm_cfg=dict(type='naiveSyncBN2d', eps=1e-3, momentum=0.01),
+        act_cfg=dict(type='ReLU'),
+        in_channels=[64, 128, 256],
+        out_channels=256,
+        start_level=0,
+        num_outs=3),
+    pts_bbox_head=dict(
+        type='Anchor3DHead',
+        num_classes=10,
+        in_channels=256,
+        feat_channels=256,
+        use_direction_classifier=True,
+        anchor_generator=dict(
+            type='AlignedAnchor3DRangeGenerator',
+            ranges=[[-50, -50, -1.8, 50, 50, -1.8]],
+            scales=[1, 2, 4],
+            sizes=[
+                [2.5981, 0.8660, 1.],  # 1.5 / sqrt(3)
+                [1.7321, 0.5774, 1.],  # 1 / sqrt(3)
+                [1., 1., 1.],
+                [0.4, 0.4, 1],
+            ],
+            custom_values=[0, 0],
+            rotations=[0, 1.57],
+            reshape_out=True),
+        assigner_per_size=False,
+        diff_rad_by_sin=True,
+        dir_offset=-0.7854,  # -pi / 4
+        bbox_coder=dict(type='DeltaXYZWLHRBBoxCoder', code_size=9),
+        loss_cls=dict(
+            type='FocalLoss',
+            use_sigmoid=True,
+            gamma=2.0,
+            alpha=0.25,
+            loss_weight=1.0),
+        loss_bbox=dict(type='SmoothL1Loss', beta=1.0 / 9.0, loss_weight=1.0),
+        loss_dir=dict(
+            type='CrossEntropyLoss', use_sigmoid=False, loss_weight=0.2)),
+    # model training and testing settings
+    train_cfg=dict(
+        pts=dict(
+            assigner=dict(
+                type='MaxIoUAssigner',
+                iou_calculator=dict(type='BboxOverlapsNearest3D'),
+                pos_iou_thr=0.6,
+                neg_iou_thr=0.3,
+                min_pos_iou=0.3,
+                ignore_iof_thr=-1),
+            allowed_border=0,
+            code_weight=[1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 0.2, 0.2],
+            pos_weight=-1,
+            debug=False)),
+    test_cfg=dict(
```

Comment:
may need to add `nms_typ` since `test_centerpoint` failed
```
_______________________________________________________________ test_centerpoint[Backend.ONNXRUNTIME] _______________________________________________________________

backend_type = <Backend.ONNXRUNTIME: 'onnxruntime'>

    @pytest.mark.parametrize('backend_type', [Backend.ONNXRUNTIME])
    def test_centerpoint(backend_type: Backend):
        from mmdeploy.codebase.mmdet3d.deploy.voxel_detection import VoxelDetection
        from mmdeploy.core import RewriterContext
        check_backend(backend_type, True)
        model = get_centerpoint()
        model.cpu().eval()
        deploy_cfg = mmcv.Config(
            dict(
                backend_config=dict(type=backend_type.value),
                onnx_config=dict(
                    input_shape=None,
                    opset_version=11,
                    input_names=['voxels', 'num_points', 'coors'],
                    output_names=['outputs']),
                codebase_config=dict(
                    type=Codebase.MMDET3D.value, task=Task.VOXEL_DETECTION.value)))
        voxeldetection = VoxelDetection(model_cfg, deploy_cfg, 'cpu')
        inputs, data = voxeldetection.create_input(
            'tests/test_codebase/test_mmdet3d/data/kitti/kitti_000008.bin')
    
        with RewriterContext(
                cfg=deploy_cfg,
                backend=deploy_cfg.backend_config.type,
                opset=deploy_cfg.onnx_config.opset_version):
            outputs = model.forward(*data)
            head = get_centerpoint_head()
            rewrite_outputs = head.get_bboxes(*[[i] for i in outputs],
>                                             inputs['img_metas'][0])

tests/test_codebase/test_mmdet3d/test_mmdet3d_models.py:159: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
mmdeploy/core/rewriters/rewriter_utils.py:371: in wrapper
    return self.func(self, *args, **kwargs)
mmdeploy/codebase/mmdet3d/models/centerpoint.py:137: in centerpoint__get_bbox
    assert self.test_cfg['nms_type'] in ['circle', 'rotate']
```